app/src/main/java/org/mozilla/rocket/privately/browse/BrowserFragment.kt

```diff
@@ -81,18 +86,19 @@ class BrowserFragment : LocaleAwareFragment(),
     private lateinit var progressView: AnimatedProgressBar
     private lateinit var siteIdentity: ImageView
 
-    private lateinit var btnLoad: ImageButton
-    private lateinit var btnNext: ImageButton
-    private lateinit var btnTracker: View
-    private lateinit var browserMenuContainer: ViewGroup
     private lateinit var toolbarRoot: ViewGroup
 
     private lateinit var trackerPopup: TrackerPopup
 
-    private var isLoading: Boolean = false
-
     private var systemVisibility = ViewUtils.SYSTEM_UI_VISIBILITY_NONE
 
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        chromeViewModel = Inject.obtainChromeViewModel(activity).apply {
+            onRestoreTabCountCompleted()
```

Comment:
why do we need to call onRestoreTabCountCompleted() here in private mode?

Reply:
since I think it's not using in private mode, so I want make the `isTabRestoredComplete` to be `ture` from the start when in the private mode

Reply:
removed the code that calling `onRestoreTabCountCompleted()` in private mode